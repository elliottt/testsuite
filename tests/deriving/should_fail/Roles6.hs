{-# LANGUAGE RoleAnnotations, TypeFamilies #-}

module Roles6 where

type family F a@R
