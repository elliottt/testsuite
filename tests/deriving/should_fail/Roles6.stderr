
Roles6.hs:5:1:
    Illegal role annotation on variable a;
    role annotations are not allowed here
    In the family declaration for ‛F’
